README.md

```diff
@@ -9,6 +9,15 @@ Cloudfuse supports clouds with an S3 or Azure interface.
 ## Installation
 Download the provided installation packages for your preferred operating system.  -- COMING SOON!
 
+### from Windows .exe installer
+
+Downloading:  
+  In order to download a prebuilt version of the software, you will need to access our [releases](https://github.com/Seagate/cloudfuse/releases) page and click the [cloudfuse_install_Windows_x86_64.exe](https://github.com/Seagate/cloudfuse/releases/download/0.2.0/cloudfuse_install_Windows_x86_64.exe) option to download the windows installer.
+
+Installing:
+	Run the downloaded .exe file as administrator and follow the installation prompts. Do not run the GUI from the installer as there is a config saving issue currently being investigated. Run the GUI separately as admin after the install completes.

```

Comment:
I think this can all be replaced by two sentences: one inviting the reader to download and install the latest release, and another to tell them to uncheck the "Launch Cloudfuse" checkbox when finishing the installation.
You could also include the fact that they should run Cloudfuse with admin rights here, although that's not essential since they're already told to run with admin privileges in the Basic Use section.